client/src/assets/js/app/typing-lesson.js

```diff
@@ -113,6 +115,22 @@
       isRunning: function() {
         return (this.state === LevelStates.Running);
       },
+      incorrectCharsSorted: function() {
```

Comment:
This method isn't named clearly.  We use it to set and sort the `this.sortedIncorrectChars`.
A better name would be `sortIncorrectChars` since it's implying that the method _does_ something vs _returns_ something.